src/pages/docs/auth/token.mdx

```diff
@@ -374,15 +379,15 @@ client, err := ably.NewRealtime(
   ably.WithAuthMethod("GET"),
   ably.WithAuthHeaders(headers),
   ably.WithAuthParams(url.Values{
-    "p1": {"param1"},
-    "p2": {"param2"},
+    [p1]() {"param1"},
```

Comment:
Need to check for instances like this where the find and replace has gone a bit rogue 🙂 (can see 3 or 4 more just on this page)